tests/PrinterTest.php

```diff
@@ -74,11 +74,18 @@ public function testTestNameCanStartOrEndWithANumber()
         $this->assertStringContainsString('✓ 123 can start or end with numbers 456', $lines[13]);
     }
 
+    public function testTestNameCanContainCapitalizedWords()
+    {
+        $lines = $this->getOutput();
+
+        $this->assertStringContainsString('✓ should preserve capitalized and partially capitalized words', $lines[14]);
```

Comment:
Shouldn't the assertion be `✓ should preserve CAPITALIZED and paRTiaLLY CAPitaLIZed words`?